application-services/custom/cloud-event/main.go

```diff
@@ -20,67 +21,64 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg/interfaces"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/models"
 	"os"
 
-	"github.com/edgexfoundry/go-mod-core-contracts/models"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
 
-	cloudevents "github.com/cloudevents/sdk-go"
-
-	"github.com/edgexfoundry/app-functions-sdk-go/appcontext"
-	"github.com/edgexfoundry/app-functions-sdk-go/appsdk"
-	"github.com/edgexfoundry/app-functions-sdk-go/pkg/transforms"
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg"
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg/transforms"
 
 	localtransforms "cloud-event/pkg/transforms"
 )
 
 const (
-	serviceKey = "cloudEventTransform"
+	serviceKey = "advancedCloudEventTransform"
```

Comment:
The service key standard we are not using is `app-` prefix. So keeping it simple I suggest `app-cloud-event`